cmd/kwil-cli/cmds/utils/parse.go

```diff
@@ -58,3 +106,34 @@ func (s *schemaDisplay) MarshalText() (text []byte, err error) {
 		return json.MarshalIndent(s.Result, "", "  ")
 	}
 }
+
+type dumpResult struct {
```

Comment:
I think #71 is conflict with this PR, and also might misinterpreted your intention.    Maybe fix #71 in this PR?